### ISSUE TITLE
Build issues on OSX

### ISSUE THREAD

**Issue by zloidemon** [tarantool/pg#12]
Can not build on OS X

```
[ 50%] Linking C shared library driver.dylib
cd /Users/vg/Devel/Tarantool/pg/build/pg && /Users/vg/Devel/CMake.app/Contents/bin/cmake -E cmake_link_script CMakeFiles/driver.dir/link.txt --verbose=1
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc   -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk -dynamiclib -Wl,-headerpad_max_install_names  -o driver.dylib -install_name /Users/vg/Devel/Tarantool/pg/build/pg/driver.dylib CMakeFiles/driver.dir/driver.c.o -lpq -rdynamic
Undefined symbols for architecture x86_64:
  "__say", referenced from:
      _pg_notice in driver.c.o
  "_coio_wait", referenced from:
      _lua_pg_connect in driver.c.o
      _pg_wait_for_result in driver.c.o
  "_fiber_is_cancelled", referenced from:
      _lua_pg_connect in driver.c.o
      _pg_wait_for_result in driver.c.o
  "_log_level", referenced from:
      _say_log_level_is_enabled in driver.c.o
  "_luaL_checkudata", referenced from:
      _lua_pg_close in driver.c.o
      _lua_pg_gc in driver.c.o
      _lua_check_pgconn in driver.c.o
  "_luaL_error", referenced from:
      _lua_pg_quote in driver.c.o
      _lua_pg_quote_ident in driver.c.o
      _lua_pg_connect in driver.c.o
      _lua_check_pgconn in driver.c.o
  "_luaL_newmetatable", referenced from:
      _luaopen_pg_driver in driver.c.o
  "_luaL_pushint64", referenced from:
      _parse_pg_value in driver.c.o
  "_luaL_register", referenced from:
      _luaopen_pg_driver in driver.c.o
  "_lua_createtable", referenced from:
      _luaopen_pg_driver in driver.c.o
      _lua_pg_execute in driver.c.o
      _safe_pg_parsetuples in driver.c.o
  "_lua_getfield", referenced from:
      _lua_pg_connect in driver.c.o
  "_lua_gettop", referenced from:
      _lua_pg_execute in driver.c.o
      _lua_pg_quote in driver.c.o
      _lua_pg_quote_ident in driver.c.o
      _lua_pg_connect in driver.c.o
  "_lua_insert", referenced from:
      _lua_push_error in driver.c.o
  "_lua_isnumber", referenced from:
      _lua_parse_param in driver.c.o
  "_lua_isstring", referenced from:
      _lua_pg_execute in driver.c.o
      _lua_pg_connect in driver.c.o
  "_lua_newuserdata", referenced from:
      _lua_pg_execute in driver.c.o
      _lua_pg_connect in driver.c.o
  "_lua_pcall", referenced from:
      _safe_pushstring in driver.c.o
      _pg_resultget in driver.c.o
  "_lua_pushboolean", referenced from:
      _lua_pg_close in driver.c.o
      _lua_pg_transaction_active in driver.c.o
      _parse_pg_value in driver.c.o
  "_lua_pushcclosure", referenced from:
      _safe_pushstring in driver.c.o
      _pg_resultget in driver.c.o
  "_lua_pushfstring", referenced from:
      _lua_pg_tostring in driver.c.o
  "_lua_pushinteger", referenced from:
      _lua_pg_execute in driver.c.o
      _lua_pg_transaction_active in driver.c.o
      _lua_pg_connect in driver.c.o
      _pg_resultget in driver.c.o
  "_lua_pushlightuserdata", referenced from:
      _safe_pushstring in driver.c.o
      _pg_resultget in driver.c.o
  "_lua_pushlstring", referenced from:
      _parse_pg_value in driver.c.o
  "_lua_pushnil", referenced from:
      _lua_pg_quote in driver.c.o
      _lua_pg_quote_ident in driver.c.o
  "_lua_pushnumber", referenced from:
      _lua_push_error in driver.c.o
      _safe_pg_parsetuples in driver.c.o
      _parse_pg_value in driver.c.o
  "_lua_pushstring", referenced from:
      _luaopen_pg_driver in driver.c.o
      _lua_pg_execute in driver.c.o
      _lua_pg_transaction_active in driver.c.o
      _pg_resultget in driver.c.o
      _save_pushstring_wrapped in driver.c.o
      _parse_pg_value in driver.c.o
  "_lua_pushvalue", referenced from:
      _luaopen_pg_driver in driver.c.o
  "_lua_setfield", referenced from:
      _luaopen_pg_driver in driver.c.o
  "_lua_setmetatable", referenced from:
      _lua_pg_connect in driver.c.o
  "_lua_settable", referenced from:
      _pg_resultget in driver.c.o
      _safe_pg_parsetuples in driver.c.o
      _parse_pg_value in driver.c.o
  "_lua_settop", referenced from:
      _luaopen_pg_driver in driver.c.o
      _parse_pg_value in driver.c.o
  "_lua_toboolean", referenced from:
      _lua_parse_param in driver.c.o
  "_lua_tolstring", referenced from:
      _lua_pg_execute in driver.c.o
      _lua_pg_quote in driver.c.o
      _lua_pg_quote_ident in driver.c.o
      _lua_pg_connect in driver.c.o
      _lua_parse_param in driver.c.o
  "_lua_tonumber", referenced from:
      _parse_pg_value in driver.c.o
  "_lua_topointer", referenced from:
      _save_pushstring_wrapped in driver.c.o
      _safe_pg_parsetuples in driver.c.o
  "_lua_type", referenced from:
      _lua_parse_param in driver.c.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [pg/driver.dylib] Error 1
make[1]: *** [pg/CMakeFiles/driver.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by rtsisyk** [comment]
Fixed by #13, please re-test

**Comment by Totktonada** [comment]
No answer during more then three years. Closing.
